osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -647,6 +658,12 @@ public override bool OnExiting(ScreenExitEvent e)
             return false;
         }
 
+        protected virtual void OnBeatmapOptionsButtonDisabledChanged(bool disabled)
+        {
+            DeleteOptionButton.Disabled = disabled;
+            ClearOptionButton.Disabled = disabled;
+        }
```

Comment:
This shouldn't exist. Simplest way to avoid this would be to define a bindable that's updated similar to this method, but is bound directly to all buttons that require a beatmap.